### ISSUE TITLE
add (link to) omemo backend implementations/libraries

### ISSUE THREAD

**Issue by azrdev** [bascht/omemo-top#92]
In addition to the client support of omemo, a list of implementing libraries would be useful for those who want to add support for a particular client (using one of said libraries). Useful attributes to track would be:
- links to website and source code
- programming language
- actively developed? (esp. w.r.t. security issues)

may omemo.top be a place for that too?

**Comment by bascht** [comment]
I could imagine having this on a separate page – probably as a plain, simple markdown page that just lists all available libraries with their specs and acts as a addressable `#anchor` for the client page.

**Comment by bascht** [comment]
@azrdev I've just merged #40 – so I'm happy for every feedback / suggestions for more + detailed information. :smile_cat:

**Comment by azrdev** [comment]
@bascht you imagine that we add the info (+ link to that separate implementation list page) which implementation is used to the new (with #94 ) client sections ?

**Comment by evaryont** [comment]
I could see this project supporting an additional subfolder of _data, called libraries. Keeping track a list of the libraries that support OMEMO would make omemo.top a useful resource to developers as well: "Here's a list of libraries to get you started on adding OMEMO support to your IM application."

**Comment by bascht** [comment]
@l29ah Yup, something along what @evaryont detailed. I don't think there are enough libraries (I guess < 10, no?) to justify moving them into data files yet. We could just add a single HTML file with all the available libraries as jump-able HTML anchors so they can be referenced from the clients.

**Comment by l29ah** [comment]
wat
